### ISSUE TITLE
Stopped working with latest ST3 dev release 3090

### ISSUE THREAD

**Issue by Fatbat** [MarioRicalde/SCSS.tmbundle#191]
With the latest dev update for ST3, 3090, you get an error when trying to open a .scss file:

"Error loading syntax file "Packages/SCSS/SCSS.tmLanguage": no such context .string-single"

Colour highlighting is broken as a result.

**Comment by bradfloodx** [comment]
+1

**Comment by trongthanh** [comment]
Same here. BTW, is there alternative / work around for now?

**Comment by Fatbat** [comment]
When you update the dev SublimeText, it puts the last version in your trash labelled something like "Sublime Text 3 (3088)". Simply drag that out of your trash back into your Program Files folder where it will sit along side the current "Sublime Text 3" folder and run ST from it instead and everything should be OK. If you emptied your trash, I'm not sure what to suggest.

However, I just looked at the ST3 dev page and it looks like they released a 3091 update yesterday that addresses the issue. Try updating if you haven't done so already.

Edit: yep, just installed 3091 and the problem appears to have been resolved.
